.storybook/main.js

```diff
@@ -1,5 +1,17 @@
+const sharedConfig = require( '../config/_shared.json' );
+const devConfig = require( '../config/development.json' );
 const storybookDefaultConfig = require( '@automattic/calypso-storybook' );
 
-module.exports = storybookDefaultConfig( {
+const storybookConfig = storybookDefaultConfig( {
 	stories: [ '../client/**/*.stories.{ts,tsx}' ],
 } );
+
+const configData = { ...sharedConfig, ...devConfig };
+storybookConfig.previewHead = ( head ) => `
+	${ head }
+	<script>
+		window.configData = ${ JSON.stringify( configData ) };
```

Comment:
This is equivalent to the following (loading calypso-config).
https://github.com/Automattic/wp-calypso/blob/96238b655502f8ea15b9486dae1e2626e7bc8f13/client/document/index.jsx#L81